core/server/data/migrations/versions/4.43/2022-04-01-10-13-add-post-newsletter-relation.js

```diff
@@ -27,14 +36,19 @@ module.exports = {
 
         logging.info(`Adding ${table}.${column} column`);
 
-        if (DatabaseInfo.isSQLite(knex)) {
-            await commands.addColumn(table, column, knex, columnDefinition);
-            return;
+        let sql = knex.schema.table(table, function (t) {
+            t.string(column, 24);
+        }).toSQL()[0].sql;
+
+        if (DatabaseInfo.isMySQL(knex)) {
+            sql += ', algorithm=copy';
```

Comment:
Do we have any guarantee that `knex` will never return the SQL string with an `;` on the end? I think this is fine, but I can't see anything in the docs about it, so would be mildly worried about this changing in a minor. 

cc @daniellockyer If you think this is fine, then so do I 👍

Reply:
I agree it would be good to guard against, _just in case_

Reply:
done ✅